code_of_conduct.md

```diff
@@ -0,0 +1,253 @@
+11. Members who are students shall abide by those academic standards established by LAN and by the college or university as a whole.
+12. In addition to this Code of Conduct and Member Disciplinary Policy, all members who attend colleges and universities are expected to abide by the policies and regulations that govern the duties, responsibilities and conduct explicitly or implicitly described in the university´s publications.
+
+**Chapter II**
+
+**Member Disciplinary Policy**
+
+The Member Disciplinary Policy explains the procedures for reporting violations, what sanctions may be imposed and the basis of which an appeal may be filed. This policy supplements and does not replace other rules of Texas Lambda Alpha Nu that may provide for disciplinary policies, rules or regulations.
+
+**Article I: Reporting Violations**
+
+- Anyone desiring to lodge a complaint, whether formal or informal, shall inform any member of the Standards Board or Vice President of Standards (referred to as VP of Standards).
+- Informal complaints are usually, but not always, solved by talking to the accused about their current behavior and encouraging them to change in the future.
+- Formal complaints will be followed by a documented report, an investigation and a hearing detailed in this document.
+
+**Article II: Distribution**
+
+All members in good standing and prospective members are entitled to receive a copy of the Code of Conduct and Member Disciplinary Policy upon request.
+
+**Article III: Definitions**
+
+- The term &quot;Fraternity&quot; shall refer to Texas Lambda Alpha Nu.
+- The term &quot;Member&quot; shall refer to those people who have been in good standing, paid all applicable dues and assessments to all units of the Fraternity and were initiated into Texas Lambda Alpha Nu.
+- The term &quot;Chapter&quot; shall refer to those separate legal entities that have a charter agreement with Texas Lambda Alpha Nu to operate as a chapter of Texas Lambda Alpha Nu. The term Chapter refers to both undergraduate and graduate entities that have an active charter with the Fraternity.
+- The term &quot;University&quot; and &quot;College&quot; means any university or college being attended by Members of Texas Lambda Alpha Nu
+- The term &quot;Student&quot; includes all persons taking courses at a university, both full-time and part-time, pursuing the baccalaureate, and/or first professional degree. Persons who are not officially enrolled for a particular term but who have a continuing relationship with a university are considered &quot;Students.&quot;
+- The term &quot;Pledge&quot; shall refer to any individual who has rushed and is on the pathway to Membership of Texas Lambda Alpha Nu.
+- The term &quot;Guest&quot; shall refer to any individual present by invitation whether personal or general at an event hosted by Texas Lambda Alpha Nu
+- The term &quot;Affiliate&quot; shall refer to any individual, Pledge or Member enjoying the friendship of Texas Lambda Alpha Nu.
```

Comment:
Oh, the way I read it, I thought "Pledge or Member" was defining "individual." I didn't realize "individual" was there to include non members. In that case, I think a comma belongs after "Pledge"